testsuite/performance/tests/src/main/java/org/keycloak/performance/dataset/DatasetLoader.java

```diff
@@ -192,11 +190,16 @@ private synchronized boolean errorReported() {
 
     private synchronized void reportError(Throwable ex) {
         logProcessedEntityCounts(true);
-        logger().error("Error occured: " + ex);
+        
+        StringWriter sw = new StringWriter();
+        ex.printStackTrace(new PrintWriter(sw));
+        
+        logger().error(String.format("Error occured in thread %s: %s", Thread.currentThread().getName(), sw.toString()));
```

Comment:
`logger().error(ex, "Error occured in thread %s", Thread.currentThread().getName());`

is probably easier one-line version of whole this change?